src/legacy/core_plugins/interpreter/common/types/kibana_datatable.js

```diff
@@ -27,7 +27,12 @@ export const kibanaDatatable = () => ({
       return {
         type: 'kibana_datatable',
         rows: context.rows,
-        columns: context.columns,
+        columns: context.columns.map(column => {
```

Comment:
Why is this change part of this PR? And also, do you still need the code on line 26?

```
context.columns.forEach(c => c.id = c.name);
```

Reply:
this was broken before, kibana_datatable requires column.id

Reply:
Right, but the code on line 26 appears to be attempting to do the same thing. Can that be removed?